bindings/azure/servicebusqueues/servicebusqueues.go

```diff
@@ -16,16 +16,13 @@ import (
 	"github.com/cenkalti/backoff/v4"
 
 	"github.com/dapr/components-contrib/bindings"
+	asbmessage "github.com/dapr/components-contrib/internal/component/azure/servicebus"
```

Comment:
What does the `internal` mean?

Reply:
In general, we have used internal to notify that the module is to be used only within components contrib repo and not directly outside. It was used for factoring out common code in the case of retry and redis client connections .... I am using the same one for factoring out common code between bindings and pubsub ASB.